engine/xml-config/src/test/resources/JaxbJobWriterTest-file4.xml

```diff
@@ -16,6 +16,9 @@
     <transformation>
         <transformer>
             <descriptor ref="Email standardizer"/>
+            <metadata-properties>
+                <property name="source">local</property>
```

Comment:
Oh I just realize that this will impact _ALL_ jobs ever made .... Could we have "local" as a default and not write it if we don't need to? Sorry for not seeing this before now.